pyproject.toml

```
@@ -126,6 +126,8 @@ src = ["earthaccess", "stubs", "tests"]
 
 [tool.ruff.lint]
 extend-select = ["I", "T20"]
+select = ["D"]
```

```suggestion
extend-select = ["I", "T20", "D"]
```

`select` will override the default enabled rules :)